packages/frontend/src/pages/profile.tsx

```diff
@@ -0,0 +1,119 @@
+import Layout from "@/components/layout/Layout";
+import React, { useEffect, type ReactElement } from "react";
+import { useUser } from "@auth0/nextjs-auth0/client";
+import ProfileUserInfo from "@/components/profile/ProfileUserInfo";
+import ProfileRecentSearch from "@/components/profile/ProfileRecentSearch";
+import {
+  // MdDarkMode,
+  // MdOutlineDarkMode,
+  MdOutlineHistory,
+} from "react-icons/md";
+import { HiOutlineStar } from "react-icons/hi";
+import ProfileNavigationBlock from "@/components/profile/ProfileNavigationBlock";
+import { TbLogout2 } from "react-icons/tb";
+import { IoMdInformationCircleOutline } from "react-icons/io";
+import { useRouter } from "next/router";
+import Head from "next/head";
+import { RiFeedbackLine } from "react-icons/ri";
+import { IoDocumentTextOutline } from "react-icons/io5";
+
+const Profile = () => {
+  const { user, isLoading } = useUser();
+  const router = useRouter();
+  // const [darkMode, setDarkMode] = useState(false);
+
+  useEffect(() => {
+    if (isLoading) return;
+
+    if (window.innerWidth > 640) {
```

Comment:
why does this redirect to `/404` when its larger than `640px`?

Reply:
bc i want it to be only accessible for phone size, for sizes that are larger than 640px, the profile will be like a pop up rather than a whole page

Reply:
then `404` is not a valid error code or a meaningful error description for this case.

The best solution is to make this page work for sizes lesser than 640px. 
**ONLY** is the case of when this page could not be customized for lesser than 640px you should instead display a meaningful error and a redirect button back.